lib/data_clean_utils.py

```diff
@@ -0,0 +1,24 @@
+import typing
+
+import pandas as pd
+
+"""
+    Cleans up all strings from each column of the NYC Taxi dataset.
+"""
+def hard_coded_cleaning_steps_mutation(df: pd.DataFrame):
+    # Correct string type confusion in vendorid.
+    df.loc[df['vendorid'] == '1', 'vendorid'] = 1
```

Comment:
`df['vendor_id'] = df['vendor_id'].astype(str)`

Reply:
Done. I converted to float first, then to int.